lint-requirements.txt

```diff
@@ -1,4 +1,4 @@
-black~=24.1
-clang-format~=17.0
-ruff~=0.1
-yamllint~=1.33
+black>=*

```

Comment:
Why `>=*` rather than simply `black`

Reply:
On second thought, we should specify a minimum version since our configs will be version specific, and that would be a good way to indicate what version we configured them for.

Reply:
Shall we set them to the latest versions?

Reply:
Sgtm